### ISSUE TITLE
Unable to get directory listing

### ISSUE THREAD

**Issue by Beakster** [microsoft/mssql-docker#475]
When I use Azure Data Studio to restore a DB into my container, if I click on the File selector, it tries to list /var/opt/mssql/data and says:
```
An error occured while loading the file browser.".
```

SQL Server logs the following error:
```
The operating system returned the error '0x80070003(The system cannot find the path specified.)' while attempting 'GetNextStream' on '(null)'.
```

This does not happen with microsoft/mssql-server-linux:latest but does happen with mcr.microsoft.com/mssql/server:2017-latest

I am running Docker version 18.09.2, build 6247962 on Windows 10 1809.

**Comment by minican** [comment]
Seems the issue was related to your current user permission , try to check if you could access both images or not .
Try to use the root account and then check it again.

**Comment by Beakster** [comment]
> Seems the issue was related to your current user permission , try to check if you could access both images or not .
> Try to use the root account and then check it again.

Thanks for the reply.  I don't see how it can be permission related.  I have given chmod 777 on the folders I'm trying to access via bash as root.  I then connect to SQL Server as SA which I assume runs as the root user.  Is there anything else I can try?

Thanks

**Comment by Beakster** [comment]
I've also tried the latest image on a kubernetes cluster in IBM cloud.  Same issue.  Changing the selected path seems to make no difference.  There is something broken with all the new images since the docker hub repo moved.  None of them can get a file listing.

![image](https://user-images.githubusercontent.com/2037159/62652199-c2d3c680-b928-11e9-8363-6deecce8d3d3.png)

**Comment by tiger5226** [comment]
Yeah I have been getting this for some time now. It looks like  it started to break after CU13. I have tested 14 and latest and the .bak import is broken with the mentioned error above. 

```
version: "3"
services:
  mssql:
    image: mcr.microsoft.com/mssql/server:2017-CU13-ubuntu
    ports:
      - "1433:1433"
    environment:
      SA_PASSWORD: "buNA8111"
      ACCEPT_EULA: "Y"
```

**Comment by Beakster** [comment]
Hi Tiger5226, good work on figuring out where the breaking change happened.  It seems that Microsoft don't care about this and there doesn't seem to be much attention paid to fixing issues with the docker image.  This is a particularly inconvenient bug for us. I guess we are at the mercy of Microsoft to fix this.  I will stay on CU13 until it is fixed.  I wonder if there is any other means to report this to MS.

**Comment by vin-yu** [comment]
Hi Everyone, I am from the SQL Server team. We do look at the issues posted in this repo - thank you for reporting this issue. We dont have a fix for this yet, and will update this issue when we have an update. Thanks.

@yualan for FYI

**Comment by jonny64bit** [comment]
> 
> 
> Hi Everyone, I am from the SQL Server team. We do look at the issues posted in this repo - thank you for reporting this issue. We dont have a fix for this yet, and will update this issue when we have an update. Thanks.
> 
> @yualan for FYI

Any update on this getting fixed? run into it today nearly 10 months later. Didn't find any 2019 version that worked ended up going back to 2017-CU13-ubuntu
